api.swagger.yaml

```diff
@@ -5338,6 +5782,48 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/manuals:read ]
+    put:
+      tags:
+        - 'manuals'
+      description: 'Update manual page information, including the root manual page.'
+      responses:
+        '200':
+          description: 'The manual page information was updated successfully.'
+        '400':
+          description: 'The supplied UUID or JSON was malformed.'
+        '403':
+          description: 'User is not authorized to update this manual page.'
+        '404':
+          description: 'The UUID does not correspond to an existing manual page.'
+        '422':
+          description: 'The manual page was not updated due to a Drupal validation failure or a JSON schema validation failure.'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/manual-page-schema-update'
+        required: true
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/manuals:write ]
+    delete:
+      tags:
+        - manuals
+      description: 'Delete a manual page, including the root manual page.'
```

Comment:
This could be misinterpreted as 2 things that will be deleted. Could we change it to:

'Delete a manual page or a manual' (assuming 'root manual page' here means the manual?)

Reply:
done